frontend/src/scenes/insights/EditorFilters/EditorFilterGroup.scss

```diff
@@ -18,7 +18,7 @@
         padding: 1em;
 
         > * + * {
-            margin-top: $default-spacing;
+            margin-top: $default_spacing;
```

Comment:
Do we also have a css var to replace this with? Should we document css variables in storybook? 🤔

Reply:
I'm really in favor of refactoring this `$default_spacing` var out as it's simply `1rem` (which looks much nicer) AND it will never make sense to change this anyway. Generally I suggest going for the `rem` unit instead, which is concise AND [practical](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px).

Reply:
I do agree but perhaps I don't do that in this already-lots-of-changes PR?

Reply:
Yeah yeah yeah, definitely no need for a sweeping refactor here